scripts/add-native-deps.js

```diff
@@ -1,29 +1,33 @@
 const fs = require('fs-extra');
-const { nativeDependencies } = require('../const');
 
 const ROOT_PACKAGE_JSON_PATH = '../../../package.json';
 const UI_PACKAGE_JSON_PATH = './package.json';
 
 function insertNativeDependencies() {
-  const rootPackageJson = fs.readJsonSync(ROOT_PACKAGE_JSON_PATH);
-  const uiPackageJson = fs.readJsonSync(UI_PACKAGE_JSON_PATH);
+  try {
+    const rootPackageJson = fs.readJsonSync(ROOT_PACKAGE_JSON_PATH);
+    const uiPackageJson = fs.readJsonSync(UI_PACKAGE_JSON_PATH);
 
-  const uiNativeDependencies = uiPackageJson.nativeDependencies;
-  const uiDependencies = uiPackageJson.dependencies;
-  const nativeDeps = uiNativeDependencies.reduce(
-    (o, dep) => ({ ...o, [dep]: uiDependencies[dep]}), {}
-  );
+    const uiNativeDependencies = uiPackageJson.nativeDependencies;
+    const uiDependencies = uiPackageJson.dependencies;
+    const nativeDeps = uiNativeDependencies.reduce(
+      (o, dep) => ({ ...o, [dep]: uiDependencies[dep] }), {}
+    );
 
-  const newPackageJson = {
-    ...rootPackageJson,
-    dependencies: {
-      ...rootPackageJson.dependencies,
-      ...nativeDeps,
-    }
-  };
+    const newPackageJson = {
+      ...rootPackageJson,
+      dependencies: {
+        ...rootPackageJson.dependencies,
+        ...nativeDeps,
+      }
+    };
 
-  fs.writeJsonSync(ROOT_PACKAGE_JSON_PATH, newPackageJson, { spaces: 2 });
-  console.log('[@shoutem/ui] - native dependencies added to root package.json');
+    fs.writeJsonSync(ROOT_PACKAGE_JSON_PATH, newPackageJson, { spaces: 2 });
+    console.log('[@shoutem/ui] - native dependencies added to root package.json');
+  } catch (e) {
+    console.log('[@shoutem/ui] - skipping adding dependencies to root package.json');
```

Comment:
Could we add actual error logging here instead of just skipping? Or is the error useless in cases of this that you've encountered?

Reply:
Roger that, will display ze error